mw-webapp/src/component/table/renderCellValue/renderCellSpan.tsx

```diff
@@ -0,0 +1,26 @@
+import styles from "src/component/table/columns.module.scss";
+
+const renderCellSpanWithValue = (value: string, isDone?: boolean) => (
+  <span className={isDone ? styles.completed : styles.notCompleted}>
+    {value}
+  </span>
+);
+
+const renderCellEmptySpan = () => (
+  <span className={styles.notCompleted}>
+    &#8203;
```

Comment:
Not transparent

Reply:
How I can make it more clear? Add comment?

Reply:
Get rid of magic number?